source/package.json

```diff
@@ -1,8 +1,7 @@
 {
   "name": "sinopia_profile_editor",
-  "description": "Editor for BIBFRAME *profiles* forked from http://github.com/lcnetdev/profile-edit",
-  "version": "0.0.1",
```

Comment:
this is the right description and version -- the one in master.

Reply:
Not sure why this fuller, more accurate description is getting stomped on.  Perhaps it needs some manual love (a commit that is squashed into this one?)